web-app/js/speciesModel.js

```diff
@@ -1,12 +1,12 @@
-
 /**
  * Manages the species data type in the output model.
  * Allows species information to be searched for and displayed.
  */
-var SpeciesViewModel = function(o, lists){
+var SpeciesViewModel = function (o, lists, populate) {
     var self = this;
-    if(!o) o = {};
-    if(!lists) lists = {};
+    if (!o) o = {};
```

Comment:
What is 'o'? Should avoid single-character variable names as they make the code harder to understand.